services/api/database/flyway/V0006__smooth_white_queen.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE "conversation_topic_mapping" (
+	"id" integer PRIMARY KEY GENERATED ALWAYS AS IDENTITY (sequence name "conversation_topic_mapping_id_seq" INCREMENT BY 1 MINVALUE 1 MAXVALUE 2147483647 START WITH 1 CACHE 1),
+	"conversation_id" integer NOT NULL,
+	"topic_id" integer NOT NULL,
+	"created_at" timestamp (0) DEFAULT now() NOT NULL,
+	CONSTRAINT "conversation_topic_mapping_unique" UNIQUE("conversation_id","topic_id")
+);
+--> statement-breakpoint
+CREATE TABLE "topic" (
+	"id" integer PRIMARY KEY GENERATED ALWAYS AS IDENTITY (sequence name "topic_id_seq" INCREMENT BY 1 MINVALUE 1 MAXVALUE 2147483647 START WITH 1 CACHE 1),
+	"code" text NOT NULL,
+	"name" text NOT NULL,
+	"description" text NOT NULL,
+	"score_weight" integer NOT NULL,
+	"created_at" timestamp (0) DEFAULT now() NOT NULL,
+	CONSTRAINT "topic_code_unique" UNIQUE("code"),
+	CONSTRAINT "topic_name_unique" UNIQUE("name"),
+	CONSTRAINT "topic_description_unique" UNIQUE("description")
+);
+--> statement-breakpoint
+CREATE TABLE "user_followed_topic" (
```

Comment:
rename to "followed_topic"

bear in mind in the future we will allow users to follow organizations

i think orgs != topics so we will probably have two tables

though we could create one unique table "follow" and put there two columns, topic_id and org_id, with the two of them cannot be non-null at the same time.